roles/reboot/tasks/main.yml

```diff
@@ -1,2 +1,6 @@
 ---
+# provide an empty iterator when a list is not provided
+# http://docs.ansible.com/ansible/playbooks_conditionals.html#loops-and-conditionals
 - include: ../../../common/ans_reboot.yaml
+  vars:
+    services: "{{ wait_for_services|default([]) }}"
```

Comment:
This one weird trick will help you avoid recursive loops...  😄

I feel like this is a dirty hack, but it *does* work